devices/Hx711/Hx711.Sample/Program.cs

```diff
@@ -11,16 +11,20 @@
 // When connecting to an ESP32 device, need to configure the SPI GPIOs
 // The following mapping is used in order to connect the WEIGTH module
 // to a M5Core device using the Grove port A
-// MOSI: connects to Grove pin 1
-// MISO: connects to Grove pin 2
+// MOSI: connects to Grove pin 1 (PD_SCK)
+// MISO: connects to Grove pin 2 (DOUT)
 // CLOCK: connect to any free port as it's not used at all
-Configuration.SetPinFunction(21, DeviceFunction.SPI1_MOSI);
-Configuration.SetPinFunction(22, DeviceFunction.SPI1_MISO);
-Configuration.SetPinFunction(23, DeviceFunction.SPI1_CLOCK);
+int spiBus = 1;
+if (spiBus == 1)
```

Comment:
Why this spiBus here?

Reply:
In sample there was declaration for spi1 pins but initialization of spi2 interface. Just make it more clear to not get exception with wrong SPI bus number selection.

Reply:
Understood. Then just fix the SPI init call. No need for this extra var here.